ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -79,8 +83,7 @@ struct MSFListCellView: View {
                 let labelAccessorySize = tokens.labelAccessorySize
 
                 if let leadingView = state.leadingView {
-                    UIViewAdapter(leadingView)
-                        .foregroundColor(Color(hasTitle ? tokens.backgroundColor : tokens.trailingItemForegroundColor))
+                    UIViewAdapter(leadingView, foregroundColor: hasTitle ? tokens.backgroundColor : tokens.trailingItemForegroundColor)
```

Comment:
Why did you stop using the foregroundColor modifier? It seems to be the right thing to do here.

Reply:
The foreground modifier appears that it doesn’t work, along with an overlay. I had to modify the uiview’s tintcolor instead for results to show

Reply:
Is it possible to retrieve the foregroundColor value and apply as the UIView tint color?
If not, creating a custom modifier for the UIViewAdapter seems like a better approach to me (instead of the initializer parameter).

Reply:
I added a custom modifier instead as I wasn't able to find a way to retrieve the foregroundColor value